### DISCUSSION TITLE
Installation Error when installing z2m

### DISCUSSION THREAD

**Discussion by Airmr** [Koenkk/zigbee2mqtt#15694] [General]
I am getting errors when installing z2m. I am a new Linux user and i dont know where to have a look for fixing this. it starts with this:
npm ERR! code 1
npm ERR! path /opt/zigbee2mqtt/node_modules/@serialport/bindings
npm ERR! command failed
npm ERR! command sh -c prebuild-install --tag-prefix @serialport/bindings@ || node-gyp rebuild
npm ERR! make: Entering directory '/opt/zigbee2mqtt/node_modules/@serialport/bindings/build'
npm ERR!   CXX(target) Release/obj.target/bindings/src/serialport.o
npm ERR! make: Leaving directory '/opt/zigbee2mqtt/node_modules/@serialport/bindings/build'
npm ERR! prebuild-install WARN install No prebuilt binaries found (target=19.3.0 runtime=node arch=arm libc= platform=linux)
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.3.0
npm ERR! gyp info using node@19.3.0 | linux | arm
npm ERR! gyp info find Python using Python version 3.9.2 found at "/usr/bin/python3"
npm ERR! gyp info spawn /usr/bin/python3

Here the log:
[2022-12-22T09_51_18_717Z-debug-0.log](https://github.com/Koenkk/zigbee2mqtt/files/10285934/2022-12-22T09_51_18_717Z-debug-0.log)

I would be glad for any tips how to fix this.
Thnx in advance
M.

**Comment by Koenkk** [comment]
Probably occurs because you are using node 19, node 18 or 16 should work. If you don't want to change the node.js version, this issue should be fixed in the next z2m release (1 jan) and in the dev branch  (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by Airmr** [reply]
Thnx. i just changed the node.js and it worked like a charm

**Reply by davidhbone** [reply]
Which version of node.js worked for you?

**Reply by Airmr** [reply]
I took 1.4
